qa/unittests/tests/conftest.py

```diff
@@ -1,7 +1,42 @@
+import uuid
+
+import boto3
+import moto.server
 import pytest
 
 pytest_plugins = [
     "pytester",
 ]
 
 pytest.register_assert_rewrite("apex_algorithm_qa_tools.scenarios")
+
+
+@pytest.fixture(scope="module")
+def moto_server() -> str:
+    """Fixture to run a mocked AWS server for testing."""
+    # Note: pass `port=0` to get a random free port.
+    # TODO avoid the private `_server` attribute https://github.com/getmoto/moto/issues/7894
```

Comment:
>  TODO avoid the private `_server` attribute https://github.com/getmoto/moto/issues/7894

that ticket has been resolved in the mean time and should be available since version 5.0.13